hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -171,7 +171,7 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
   logic first_error;
   assign first_error = ~reg2hw.error_log.valid.q & racl_error_arb.valid;
 
-  // Writing 1 to the error valid bit clears the log again
+  // Writing 1 to the error valid bit clears the log and log address again
```

Comment:
This should be added to the description of the `valid` field in the `ERROR_LOG` register, too (e.g., "Writing a one clears this register and the !!ERROR_LOG_ADDRESS register.").